internal/pkg/manifest/validate.go

```diff
@@ -1411,6 +1412,9 @@ func (p PublishConfig) validate() error {
 
 // validate returns nil if Topic is configured correctly.
 func (t Topic) validate() error {
+	if t.Type != nil && !contains(aws.StringValue(t.Type), validTopicsTypeValues) {
```

Comment:
```suggestion
	if !contains(aws.StringValue(t.Type), validTopicsTypeValues) {
```

Reply:
Actually that check is required as we allow customers to not define type and we set default value to it. In that case type is nil and that is fine. But if it is not nil then we check if it has valid values or not. Does that make sense?